### ISSUE TITLE
Extension doesn't works on Flatpak version

### ISSUE THREAD

**Issue by Flashwalker** [ungoogled-software/ungoogled-chromium#2102]
### OS/Platform

Debian, Ubuntu, and derivatives

### Installed

Flatpak

### Version

105, 106

### Have you tested that this is not an upstream issue or an issue with your configuration?

- [X] I have tried reproducing this issue in Chrome and it could not be reproduced there
- [X] I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
- [ ] I have tried reproducing this issue in ungoogled-chromium with a new and empty profile and it could not be reproduced there

### Description

[This proxy extension](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch) doesn't works on Flatpak version properly. Doesn't proxyfying.

### How to Reproduce?

1. Come to Russia (you'll need a russian IP)
2. Install **Ungoogled-Chromium** via **Flatpak**
3. Install extension [Обход блокировок рунета](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch)
4. Try to open e.g. http://hdrezka.tv/ which is bloked by russian ISPs
5. If extension works you will get access to this site. But it doesn't works on **Ungoogled-Chromium** from **Flatpak**.

### Actual behaviour

[Extension](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch) doesn't proxyfying on **Ungoogled-Chromium** installed via **Flatpak**. So blocked sites are inaccessible

### Expected behaviour

When [Extension](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch) works you will get access to the blocked site.

### Additional context

[Extension](https://chrome.google.com/webstore/detail/%D0%BE%D0%B1%D1%85%D0%BE%D0%B4-%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%D1%80%D0%BE%D0%B2%D0%BE%D0%BA-%D1%80%D1%83%D0%BD%D0%B5%D1%82%D0%B0/npgcnondjocldhldegnakemclmfkngch) works properly on **Ungoogled-Chromium** from **.deb** package and from **AppImage** package

**Comment by Flashwalker** [comment]
https://github.com/anticensority/runet-censorship-bypass/issues/158

**Comment by Flashwalker** [comment]
This extension uses a proxy .pac file.
I found that:
 If I try a another proxy extension, the proxying works. As long as I don't set proxying through the .pac file in the extension settings. Coonection resets each time i set the .pac file in settings.

 This affects the version of **Ungoogle-Chromium** installed by flatpak.
 
 I've tried another browser installed by flatpak, which is based on Firefox, and the .pac file works there.
 
So it looks like **Ungoogle-Chromium** flatpak issue.
